### PR TITLE
Tutorial gl+rnn

### PR DESCRIPTION
**Pull request by luisaforozco**
In this PR:
- Added a tutorial: jupyter notebook showing how to use RNN in the workflow of GL calibration.
- We could consider creating some observation data ("experimental data" that we read from a file) instead of preparing synthetic data in the tutorial. That might make the tutorial  a bit clearer.
- I would also like some feedback from @chyalexcheng how difficult/unclear is this tutorial? I could also change the fact that the grainlearning part is in a function to separate blocks.
- Added documentation about this same process.
- Modify `rnn/predict.py` and `rnn/windows.py` because I found that `tf.Data.Dataset.map` is extremely slow (10X). Now I unpack the dataset before calling `windows.predict_over_windows`. This also has the advantage of making the code more readable and understandable.
- Also updated the pre-trained models to be consistent with the fact that before we were not really adding e0 to `contact_params`.

### CODE REVIEW

File: grainlearning/rnn/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ During the installation of grainLearning activate extra `rnn`:
 
 `poetry install --extras "rnn"` or `pip install .[rnn]`
 
-*Note:* This will install a version of tensorflow depending on your system. If gpu dependencies are not installed or activated, it will default to CPU version. Check the installation of tensorflow and/or re-install it following specific instructions for your hardware.
+*Note:* This will install a version of tensorflow depending on your system. If gpu dependencies are not installed or activated, it will default to **CPU** version. Check the installation of tensorflow and/or re-install it following specific instructions for your hardware.
```
Comment (chyalexcheng) [inline grainlearning/rnn/README.md:14]: Add a link on the installation of GPU dependencies?

File: grainlearning/rnn/trained_models/rnn_triaxial_drained/config.yml
Comment (chyalexcheng) [inline grainlearning/rnn/trained_models/rnn_triaxial_drained/config.yml:1]: Will like to take a closer look into this config file.

**Review by chyalexcheng — COMMENTED** [review]
How is the GPU dependencies of tensorflow handled by poetry?

File: grainlearning/rnn/README.md (hunk 1 above)
Comment (luisaforozco) [reply grainlearning/rnn/README.md:14]: I thought about it but tensorflow changes such instructions often, and they have many different possibilities thus, I think is best just to leave it to the user, otherwise they are going to come to us: the specific set of instructions that you suggested don't work anymore... 
There are enough blogpposts on the internet explaining how to install tensorflow for specific hardware.

Comment (luisaforozco) [reply grainlearning/rnn/README.md:14]: Replying to the last comment: GPU dependencies are not handled by poetry, only CPU version of tensorflow is installed. That is why such piece of information is on the README.

**Review by chyalexcheng — APPROVED** [review]
